hack/community/generate.sh

```diff
@@ -1,40 +1,28 @@
 #!/bin/bash
 
-# Given the current WMCO manifests, generate new community manifests.
-
-# Must be ran within WMCO root.
+# Given the current community WMCO manifests, generate new community manifests
+# in the ARTIFACT_DIR.
 
 # Example:
-# Cut community v5.1.0 for community-4.10
-# Run: bash ./hack/community/generate.sh 5.1.0 community-4.10 community-4.10-hash ../community-operators-prod
+# Run: bash ./hack/community/generate.sh WMCO_VERSION ARTIFACT_DIR
```

Comment:
renaming `ARTIFACT_DIR` to `OUTPUT_DIR` would make the parameter more generalized and easier to understand, especially for someone not familiar with openshift CI.

Reply:
Applies to all references in this script